src/renderer/viz/expressions/basic/variable.js

```diff
@@ -67,14 +67,18 @@ export default function variable (name) {
             } else {
                 return false;
             }
-            // Indicate success
+
             return true;
         },
 
         get: (obj, prop) => {
             switch (prop) {
                 case 'parent':
                 case 'notify':
+                case 'propertyName':
+                    return alias.propertyName;
+                case 'name':
+                    return alias._dimension ? alias.propertyName : alias.name;
```

Comment:
**Note**: I've opened an issue about that here https://github.com/CartoDB/carto-vl/pull/1281